### DISCUSSION TITLE
Plugin for p5.js

### DISCUSSION THREAD

**Discussion by torinmb** [shader-park/shader-park-core#65] [Ideas]
P5.js has a webgl mode(webgl1) and we could create a plugin that would allow people to map a Shader Park sculpture to a piece of geometry similar to how things work in three.js

**Comment by torinmb** [comment]
This relies on #67

**Comment by torinmb** [comment]
Found an example that enables webgl2 in P5
https://editor.p5js.org/a_/sketches/2zXozr2NJ

**Reply by torinmb** [reply]
It's not the cleanest, but it works :)

**Comment by PWhiddy** [comment]
I tried this same code snippet but wasn’t able to get it working yet

**Comment by torinmb** [comment]
https://github.com/processing/p5.js/issues/2536#issuecomment-1074448308 
we'll see if we get any response. I might need to submit it as a separate issue
